workbench/config/WorkbenchConfig.php

```diff
@@ -60,17 +60,23 @@ function __construct() {
             $envKey = str_replace("___DOT___", ".", $envKey);
         
             $envKeyParts = explode($configDelim, $envKey);
-        
+
+            $lastKey = end($envKeyParts);
+            reset($envKeyParts);
```

Comment:
@rvwatch: Could you explain why we use this way to get the end of the array (get the end then reset the array)? Is this idiomatic PHP?

Reply:
I've been reading some more on this. Though end()/reset() is a relatively common pattern, I don't like the idea of messing with the internal pointer of the original array and having to reset it. That feels flimsy. I think switching it to something like: `end((array_values($array)))` may be a better solution as it doesn't touch the pointer of the original array. No need to reset and it should be fine performance wise as it only hits small arrays. 

I also found this interesting stack overflow comment if you're curious. https://stackoverflow.com/a/41795859